pkgs/development/compilers/mono/default.nix

```diff
@@ -0,0 +1,120 @@
+{ stdenv, lib, fetchurl, fetchpatch, callPackage, cmake, bison, pkgconfig, perl, python2
+, gettext, ncurses, zlib, cacert
+, Foundation, libobjc
+, withX11 ? true, libgdiplus, libX11, libXinerama, cairo, glib, gtk2, gdk_pixbuf, librsvg
+, withCUPS ? true, cups
+, withALSA ? true, alsaLib
+, withLLVM ? true
+}:
```

Comment:
I see, thank you for double-checking. Fixed, also made it so that GTK and friends aren't used on Darwin -- does that sound right?